app/controllers/tmdb_controller.rb

```diff
@@ -81,8 +81,7 @@ def two_actor_search
   end
 
   def actor_more
-    @actor_id = params[:actor_id]
-    tmdb_handler_actor_more(@actor_id)
+    @actor = tmdb_handler_person_detail_search(params[:actor_id])
```

Comment:
The `tmdb_handler_actor_more` id just a wrapper around `tmdb_handler_person_detail_search`. Nothing else calls it. I got rid of it.